### DISCUSSION TITLE
Resume_Review < Garima Jain>

### DISCUSSION THREAD

**Discussion by Garimatech19** [recodehive/resume-pitch#93] [Q&A] [answered]
My resume is -
[GarimaJain(Resume) final (1).pdf](https://github.com/user-attachments/files/17006181/GarimaJain.Resume.final.1.pdf)
Currently I'm looking for Software Engineer Intern as I'm 4th year student.

**Comment by sanjay-kv** [comment, marked as answer]
I didnt see any topmate request for this 
could you check that again: 
if you wanna book again: https://topmate.io/sanjaykv/1047284/pay
use the same freecode you used before.

**Comment by Garimatech19** [comment]
hello sir, I want to that how to answer a question according to STAR method , if i had never face any kind of situation in the past

**Reply by PranjalTrivedi** [reply]
Hi Garima, I will explain you
Every technical interview has STAR method questions.
S - Situation
T - Task
A - Action 
R - Results

This means tell them a example of experience, you faced any SITUATION and its TASK. how you solved that task by which ACTION and what are the RESULTS came out.
